src/screens/VaultScreen.js

```diff
@@ -30,17 +39,45 @@ export default class VaultScreen extends React.Component {
           upvotes: 0,
           category: 2,
           id: 1,
+          veteran_has_upvoted: boolean
         },
         */
-      ]
+      ],
+      stillLoading: true,
     };
     this.falseState = this.falseState.bind(this);
     this.setOpposite = this.setOpposite.bind(this);
     this.renderResourceContent = this.renderResourceContent.bind(this);
   }
 
   componentDidMount() {
+    this.retrieveResources();
+  }
 
+  retrieveResources() {
+    ResourceRequester.resources().then((response) => {
+      data = response.map(function(item) {
+        return {
+          id: item.id,
+          title: item.file_name,
+          date: this.formatDate(item.updated_at),
+          link: item.url,
+          partner_org: item.owner_id,
+          description: item.description,
+          category: this.getCategory(parseInt(item.category)),
+          upvotes: item.num_upvotes,
+          veteran_has_upvoted: item.veteran_has_upvoted,
+        };
```

Comment:
Why are you reserializing here?

Reply:
@clairelin135 why is this being serialized again on the frontend?

Reply:
the resources are sent in an array in json form and i need to retrieve the attributes in each field